information-schema/information-schema-tiflash-replica.md

```diff
@@ -28,3 +28,13 @@ DESC tiflash_replica;
 +-----------------+-------------+------+------+---------+-------+
 7 rows in set (0.01 sec)
 ```
+
```

Comment:
quick-start-with-htap.md L165
```
- `PROGRESS` means the progress of the replication. The value is between 0.0 and 1.0. 1 means at least one TiFlash replica is replicated.
```
[TiDB Cloud HTAP Quick Start](https://docs.pingcap.com/tidbcloud/tidb-cloud-htap-quickstart#step-3-compare-the-query-performance-between-row-based-storage-and-columnar-storage:~:text=PROGRESS%20means%20the%20progress%20of%20the%20replication.%20The%20value%20is%20between%200%20and%201.%201%20means%20at%20least%20one%20replica%20is%20replicated.) might need to be updated.

Reply:
Done.